kinder/pkg/cluster/manager/actions/kubeadm-config.go

```diff
@@ -181,9 +183,10 @@ func getKubeadmConfig(c *status.Cluster, n *status.Node, data kubeadm.ConfigData
 	if err != nil {
 		return "", err
 	}
+	log.Debugf("kubeadm version %s", kubeadmVersion)
```

Comment:
e.g This could be a good place to validate the kubeadm version actually supports the requested config version.
Also, we should make sure that if the config version is not specified, kinder uses a default config version like it is happening today